core/src/test/java/apoc/trigger/TriggerDisabledTest.java

```diff
@@ -65,4 +65,28 @@ public void testTriggerDisabledResume() {
     public void testTriggerDisabledPause() {
         db.executeTransactionally("CALL apoc.trigger.pause('test-trigger')");
     }
+
+    @Test
+    public void testTriggerDisabledInstall() {
+        db.executeTransactionally("CALL apoc.trigger.install('neo4j', 'test-trigger', 'RETURN 1', {phase: 'before'})");
+    }
+    @Test
+    public void testTriggerDisabledDrop() {
+        db.executeTransactionally("CALL apoc.trigger.drop('neo4j', 'test-trigger')");
+    }
+    
+    @Test
+    public void testTriggerDisabledDropAll() {
+        db.executeTransactionally("CALL apoc.trigger.dropAll('neo4j')");
+    }
+
+    @Test
+    public void testTriggerDisabledStart() {
+        db.executeTransactionally("CALL apoc.trigger.start('neo4j', 'test-trigger')");
+    }
```

Comment:
What is this testing? It seems like we get no failure at all if triggers are disabled? Is that normal?

Reply:
Yes it's a wanted behavior, because this tests actually fail but are catched by the `exceptionRule.expectMessage(TriggerUtils.NOT_ENABLED_ERROR);` (line 40). In fact, with `apocConfig().setProperty(APOC_TRIGGER_ENABLED, true);` they fails with error `Expected test to throw`.